README.md

```diff
@@ -1,38 +1,76 @@
-# Newdle
+## WARNING: newdle is not yet production ready
+### Currently missing
+ * [ ] E-mail sending;
+ * [ ] Integration with calendaring systems;
+ * [ ] Integration with Indico;
 
-This is something new and cool! 🎉
+If you would like to help with any of that, please contact us through the relevant GitHub issue.
 
-## Database
+# Newdle [![Travis Build Status](https://travis-ci.com/indico/newdle.svg?branch=master)](https://travis-ci.org/indico/newdle) [![License](https://img.shields.io/github/license/indico/newdle.svg)](https://github.com/indico/newdle/blob/master/LICENSE) [![Made at CERN!](https://img.shields.io/badge/CERN-Open%20Source-%232980b9.svg)](https://home.cern)
+
+<img src="./newdle/client/src/images/logo.svg" width="300"><br>
+
+
+**Newdle** is the new, shiny tool brought to you by the [Indico Team](https://getindico.io/about) @ [CERN](https://home.cern) 🎉
+Here at CERN we **hold a lot of meetings**. While [Indico](https://getindico.io) makes it super easy to manage those meetings, we still **lose a lot of time** 📈 trying to schedule them, which usually involves numerous emails and private messages. That is what newdle has been created for: to streamline the process of choosing **the perfect date and time** 🗓 for your next meeting/event.
+
+Newdle is part of the [MALT project](https://malt.web.cern.ch).
+
+## Why another tool?
+
+It is true that there are already several commercial and Open Source solutions available that facilitate creating "polls". One of them is [Doodle](https://doodle.com) which was one of our main inspirations when starting the development process. The biggest advantage of newdle over it is the fact that newdle is free of charge and Open Source - you just have to set it up on your server and you can start newdling! Apart from the commercial products on the market, there is a handful of Open Source projects e.g. [Nuages](https://nuages.domainepublic.net) or [croodle](https://github.com/jelhan/croodle) which differ a lot in the features they offer. We have noticed that the majority of those tools don't provide users with proper User Experience which makes using them harder than it should be. This is one of the major advantages of newdle. We made sure that our tool is easy to use and intuitive with really good-looking design. Since we are also the developers behind [Indico](https://getindico.io) we thought of integrating newdle with it. So, for example after a poll/newdle has finished it will be possible to create an event in [Indico](https://getindico.io). Also newdle will make it a lot easier to choose possible slots for your meeting since we will integrate it with enterprise calendars to fetch participant availability. While making sure we still protect your data.
```

Comment:
```suggestion
It is true that there are already several commercial and Open Source solutions available that facilitate creating "polls". One of them is [Doodle](https://doodle.com) which was one of our main inspirations when starting the development process. The biggest advantage of newdle over it is the fact that newdle is free of charge and Open Source - you just have to set it up on your server and you can start newdling! Apart from the commercial products on the market, there is a handful of Open Source projects e.g. [Nuages](https://nuages.domainepublic.net) or [croodle](https://github.com/jelhan/croodle) which differ a lot in the features they offer. We have noticed that the majority of those tools don't provide users with proper User Experience which makes using them harder than it should be. This is one of the major advantages of newdle. We made sure that our tool is easy to use and intuitive with really good-looking design.

## Integration
**This is still ongoing work**
Since we are also the developers behind [Indico](https://getindico.io) we thought of integrating newdle with it. So, for example after a poll/newdle has finished it will be possible to create an event in [Indico](https://getindico.io). Also newdle will make it a lot easier to choose possible slots for your meeting since we will integrate it with enterprise calendars to fetch participant availability, while making sure we still protect your data.
```

Reply:
I would just split the last part into a different section. Otherwise it looks perfect to me, thanks!